src/AccessibilityInsights.SharedUxTests/Resources/ConfigSettings.json

```diff
@@ -1,6 +1,7 @@
 {
     "AlwaysOnTop": false,
     "AppVersion": "1.1.700.1",
+    "ColorContrastVersionThresholdSize": 20000,
```

Comment:
Should this be:

```suggestion
    "ColorContrastVersionThresholdSize": 10000,
```

Reply:
This is intentionally not 10000. This file includes _non-default_ values as a proof that the file loading code is properly handling the JSON.